13/umbraco-deploy/deploy-settings.md

```diff
@@ -60,13 +59,14 @@ For illustration purposes, the following structure represents the full set of op
             "ExportMemberGroups": true,
             "ReloadMemoryCacheFollowingDiskReadOperation": false,
             "AllowDomainsDeploymentOperations": "None",
+            "AllowWebhooksDeploymentOperations": "None",
+            "TrashedContentDeploymentOperations": "Import",
             "PreferLocalDbConnectionString": false,
             "MediaFileChecksumCalculationMethod": "PartialFileContents",
             "NumberOfSignaturesToUseAllRelationCache": 100,
             "ContinueOnMediaFilePathTooLongException": false,
             "SuppressCacheRefresherNotifications": false,
-            "HideConfigurationDetails": false,
-            "AllowWebhooksDeploymentOperations": "None"
+            "HideConfigurationDetails": false
```

Comment:
this change has been added in PR https://github.com/umbraco/UmbracoDocs/pull/5750